config/config.go

```diff
@@ -25,6 +25,7 @@ type Config struct {
 	ContentPublishedTopic      string        `envconfig:"KAFKA_CONTENT_PUBLISHED_TOPIC"`
 	KafkaProducerTopic         string        `envconfig:"KAFKA_PRODUCER_TOPIC"`
 	ZebedeeURL                 string        `envconfig:"ZEBEDEE_URL"`
+	KeywordsLimit              string        `envconfig:"KEYWORDS_LIMITS"`
```

Comment:
This variable should be an integer type and then you wouldn't need to convert it in the handle method in handler.go.

```suggestion
	KeywordsLimit              int           `envconfig:"KEYWORDS_LIMITS"`
```